addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -1047,13 +1047,12 @@ export class SelectionPlugin extends Plugin {
     }
 
     focusEditable() {
-        if (this.editable.contains(this.document.activeElement)) {
+        const { editableSelection, documentSelectionIsInEditable } = this.getSelectionData();
+        if (this.editable.contains(this.document.activeElement) && documentSelectionIsInEditable) {
```

Comment:
Ok for 18.0 but from saas-18.4 I think you'll probably need to use `currentSelectionIsInEditable` to handle website and the iframe 🤔